[Helm] Makefile does not work with helm v3

**Describe the bug**
```
$ make helm
rm -f production/helm/*/requirements.lock
Error: unknown command "init" for "helm"

Did you mean this?
	lint

Run 'helm --help' for usage.
make: *** [Makefile:341: helm] Error 1
```

```
$ helm version
version.BuildInfo{Version:"v3.3.4", GitCommit:"a61ce5633af99708171414353ed49547cf05013d", GitTreeState:"clean", GoVersion:"go1.14.9"}
```

`init` is removed from helm v3: https://github.com/helm/helm/issues/6996

**To Reproduce**
Steps to reproduce the behavior:
1. Install helm v3
2. Call `make helm`

**Expected behavior**
helm will lint and package the charts.

**Environment:**
 - Infrastructure: laptop
 - Deployment tool: helm

--- a/Makefile
+++ b/Makefile
@@ -339,7 +339,6 @@ helm: PACKAGE_ARGS ?=
 helm:
 	-rm -f production/helm/*/requirements.lock
 	@set -e; \
-	helm init -c; \
 	helm repo add elastic https://helm.elastic.co ; \
 	helm repo add grafana https://grafana.github.io/helm-charts ; \
 	helm repo add prometheus https://prometheus-community.github.io/helm-charts ; \
@@ -351,8 +350,6 @@ helm:
 	rm -f production/helm/*/requirements.lock
 
 helm-install:
-	kubectl apply -f tools/helm.yaml
-	helm init --wait --service-account helm --upgrade
 	HELM_ARGS="$(HELM_ARGS)" $(MAKE) helm-upgrade
 
 helm-install-fluent-bit:

